File: codesLeftInWindows/CP-Problems-tilnoeneBackup/URI/1318.py
while True:
    l = input().split()
    n = int(l[0])
    m = int(l[1])
    
    if n == m == 0: break
    
    v = []
    for i in range(10010):
        v.append(0)
    
    linha = [int(i) for i in input().split()]
    
    rep = 0
    for i in linha:
        v[i] += 1
        
    for i in v:
        if(i > 1):
            rep += 1
    
    print(rep)

File: codesLeftInWindows/CP-Problems-tilnoeneBackup/URI/1318.cpp
#include <iostream>

using namespace std;

int main(){
    int n, m, t;
    
    while(true){
        cin >> n >> m;
        if(n == 0 && m == 0) break;
        
        int v[10010] = {}, rep = 0;
        
        for(int i = 0; i < m; i++){
            cin >> t;
            v[t]++;
        }
        
        for(int i = 0; i < 10010; i++){
            if(v[i] > 1){
                rep++;
            }
        }
        
        cout << rep << endl;
    }
    
    return 0;
}